test-suite/src/test/scala/scala/scalajs/testsuite/javalib/RegexTest.scala

```diff
@@ -52,6 +52,23 @@ object RegexTest extends JasmineTest {
       split("", "\n", Array(""))
       split("", "", Array(""))
 
+      // Should remove leading empty match under some conditions - #1171
+      // These tests are "measured" on the JVM since the spec is unclear
+      split("abc", "(?=a)", Array("abc"))
+      split("abc", "(?=b)", Array("a", "bc"))
+      split("abc", "(?=a)|b", Array("", "a", "c"))
+      split("abc", "", Array("", "a", "b", "c"))
+      split("abc", "(?=a)|(?=b)", Array("", "a", "bc"))
```

Comment:
Well, they now have specifically said it was an intended behavior change.
Bug reported as #2573